spec/models/snippet_spec.rb

```diff
@@ -25,8 +25,8 @@
     end
 
     it "has a user_id" do
-      user = FactoryGirl.build_stubbed(:user)
-      snippet = FactoryGirl.build_stubbed(:snippet) 
+      user = FactoryGirl.build_stubbed(:user, id: 2)
```

Comment:
I would write this spec as:

```ruby
it "belongs to a user"
user = FactoryGirl.build_stubbed(:user)
snippet = FactoryGirl.build_stubbed(:snippet, user: user)

expect(snippet.user).to eq(user)
```

It's dangerous to set an id explicitly, since if 🏭  👧 or anything else that's auto-incrementing tries to set the same id, ActiveRecord will throw an error

I recommend you use the [shoulda matchers](https://github.com/thoughtbot/shoulda-matchers) gem to test associations